Extensions/xkit_preferences.js

```diff
@@ -59,10 +59,16 @@ XKit.extensions.xkit_preferences = new Object({
 			XKit.tools.add_css(mobile_control_panel, 'mobile_xkit_menu');
 		}
 
-		$("#account_button").before(m_html);
-		$("#account_button > button").attr("tabindex", "8");
-		$("header div div:has([aria-label='Account'])").before(m_html);
-		$(".xkit--react #xkit_button").attr('tabindex', '0');
+		if (XKit.page.react) {
+			XKit.interface.translate("Account").then(AccountLabel => {
```

Comment:
Minor nitpick - camelCase instead of PascalCase

Reply:
using `accountLabel` doesn't really make sense since the string we're using/translating is literally `Account` with the capital A. i'd rather drop the Label part if anything

Reply:
fair enough

Reply:
:-1: